Add example PostgreSQL config w/env vars (#4031)
sensu/sensu-docs: content/sensu-go/6.8/operations/deploy-sensu/datastore.md

To update your Sensu PostgreSQL configuration, repeat the `sensuctl create` process.
You can expect PostgreSQL status updates in the [Sensu backend logs][2] at the `warn` log level and PostgreSQL error messages in the [Sensu backend logs][2] at the `error` log level.

#### Use environment variables to configure PostgreSQL

The Sensu backend uses the [libpq][23] to make connections to PostgreSQL.
libpq supports a number of [environment variables][21] that can be injected into the PostgreSQL data source name (DSN).
Sensu loads these environment variables at runtime using the system's environment variable file.

You can use the libpq environment variables to connect to PostgreSQL without exposing sensitive information, like usernames and passwords, in your PostgreSQL configuration.
To do this, [define the libpq environment variables][22] as described in the backend reference.

> The client automatically looks up these environment variables, so you do not need to reference them in your PostgresConfig definition.

Author: Is Sensu "the client" in this case?

Reviewer: Yes, it is :)

> The client automatically looks up these environment variables, so you do not need to reference them in your PostgresConfig definition.

Author: ```suggestion
Sensu automatically looks up these environment variables, so you do not need to reference them in your PostgresConfig definition.
```

For example, to use libpq environment variables to define the PostgreSQL username, password, port, and database name in the [Sensu backend environment file][22]:

{{< code shell >}}
PGUSER=<PostgreSQL_username>
PGPASSWORD=<PostgreSQL_password>
PGPORT=5432
PGDATABASE=sensu_events
{{< /code >}}

With these environment variables defined, the PostgreSQL configuration does not need to include the username, password, port, or database name:

{{< language-toggle >}}

{{< code yml >}}
---
type: PostgresConfig
api_version: store/v1
metadata:
  name: postgres_datastore
spec:

>   dsn: "postgresql://postgres.example.com?sslmode=disable"

Reviewer: ```suggestion
  dsn: "postgresql://postgres.example.com"
```

  pool_size: 20
  strict: true
{{< /code >}}

{{< code json >}}
{
  "type": "PostgresConfig",
  "api_version": "store/v1",
  "metadata": {
    "name": "postgres_datastore"
  },
  "spec": {

>     "dsn": "postgresql://postgres.example.com?sslmode=disable",

Reviewer: ```suggestion
    "dsn": "postgresql://postgres.example.com",
```
